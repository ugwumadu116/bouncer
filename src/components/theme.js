import * as colors from "./color/color";
import * as fontsize from "./text/fontsize";
import * as fontweight from "./text/fontweight";
import * as lineheight from "./text/lineheight";
import * as letterspacing from "./text/letterspacing";
import * as btnSizes from "./button/button";
import * as btnState from "./button/btnState";
import * as inputsizes from './form/input/inputsizes'
import * as inputbuttonSize from './form/inputButton/inputbuttonsizes'
import * as selectSize from './form/select/selectSizes'


const theme = {
  colors,
  fontsize,
  fontweight,
  lineheight,
  letterspacing,
  btnSizes,
  btnState,
  inputsizes,
  inputbuttonSize,
  selectSize
};
export default theme;
