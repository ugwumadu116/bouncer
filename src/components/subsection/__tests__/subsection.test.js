import React from 'react'
import renderer from 'react-test-renderer'
import {ThemeProvider} from 'styled-components'

import theme from '../../../components/theme'
import Subsection from '../index'



it('renders correctly', ()=>{
    const tree = renderer.create(
    <ThemeProvider theme={theme}>
      <Subsection/>  
    </ThemeProvider>
    
    
    ).toJSON();
    expect(tree).toMatchSnapshot();
})