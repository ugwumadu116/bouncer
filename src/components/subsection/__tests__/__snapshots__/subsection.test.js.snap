// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa lauwRn"
>
  <div
    className="text-align"
  >
    <p
      className="sc-bwzfXH koyxRV"
    />
  </div>
  <div
    className="d-flex"
  >
    <div
      className="d-flex"
    >
      <div>
        <img
          alt="laptop"
          src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575976812/bouncer-frontend/tv_tray_c0ht3d.png"
        />
      </div>
      <div
        className="sc-ifAKCX dljjaB p-3"
      >
        <p
          className="sc-bwzfXH ejpdlN"
        >
          01 Jan, 2015
        </p>
        <p
          className="sc-bwzfXH jTrSdS"
        >
          Typesetting industry
        </p>
        <p
          className="sc-bwzfXH ckPPLG"
        >
          Lorem Ipsum is simply dummy text of the printing and
        </p>
      </div>
    </div>
    <div
      className="d-flex"
    >
      <div>
        <img
          alt="laptop"
          src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575976812/bouncer-frontend/tv_tray_c0ht3d.png"
        />
      </div>
      <div
        className="sc-ifAKCX dljjaB p-3"
      >
        <p
          className="sc-bwzfXH ejpdlN"
        >
          01 Jan, 2015
        </p>
        <p
          className="sc-bwzfXH jTrSdS"
        >
          Typesetting industry
        </p>
        <p
          className="sc-bwzfXH ckPPLG"
        >
          Lorem Ipsum is simply dummy text of the printing and
        </p>
      </div>
    </div>
    <div
      className="d-flex"
    >
      <div>
        <img
          alt="laptop"
          src="https://res.cloudinary.com/decagonbouncer/image/upload/v1575976812/bouncer-frontend/tv_tray_c0ht3d.png"
        />
      </div>
      <div
        className="sc-ifAKCX dljjaB p-3"
      >
        <p
          className="sc-bwzfXH ejpdlN"
        >
          01 Jan, 2015
        </p>
        <p
          className="sc-bwzfXH jTrSdS"
        >
          Typesetting industry
        </p>
        <p
          className="sc-bwzfXH ckPPLG"
        >
          Lorem Ipsum is simply dummy text of the printing and
        </p>
      </div>
    </div>
  </div>
</div>
`;
